be/src/http/action/stop_be_action.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "http/action/stop_be_action.h"
+
+#include <sstream>
+#include <string>
+
+#include "http/http_channel.h"
+#include "http/http_request.h"
+#include "http/http_status.h"
+#include "util/defer_op.h"
+
+namespace starrocks {
+extern std::atomic<bool> k_starrocks_exit_quick;
+
+std::string StopBeAction::construct_response_message(std::string msg) {
+    std::stringstream ss;
+    ss << "{";
```

Comment:
the constructed message is not a valid json format, intended?

Reply:
No, i will change this